packages/sheets-formula-ui/src/services/render-services/ref-selections.render-service.ts

```diff
@@ -44,13 +44,15 @@ export class RefSelectionsRenderService extends BaseSelectionRenderService imple
         @Inject(ThemeService) themeService: ThemeService,
         @IShortcutService shortcutService: IShortcutService,
         @Inject(SheetSkeletonManagerService) sheetSkeletonManagerService: SheetSkeletonManagerService,
+        @IContextService private readonly _contextService: IContextService,
```

Comment:
It should be a `protected` value in the parent class please you pass it to the `super`.